personalize.admin.campaign.inc

```diff
@@ -161,17 +161,17 @@ function personalize_campaign_wizard_process_bar($form, &$form_state, $agent_dat
     ),
   );
   if (!empty($agent_data->machine_name)) {
-    // Save the current subform.
-    $form['process_bar']['actions']['save'] = array(
-      '#type' => 'submit',
-      '#value' => t('Save'),
-    );
-
     // Change campaign status.
     $form['process_bar']['actions']['status'] = personalize_status_change_form($form, $form_state, $agent_data);
     // Assign the submit handler to the button to bypass full form submission.
     $form_state['build_info']['files'][] = drupal_get_path('module', 'personalize') . '/personalize.admin.inc';
     $form['process_bar']['actions']['status']['status_wrapper']['submit']['#submit'] = array('personalize_status_change_form_submit');
+
+    // Save the current subform.
+    $form['process_bar']['actions']['save'] = array(
+      '#type' => 'submit',
+      '#value' => t('Save'),
+    );
   }
```

Comment:
Rotated the "status" button and the "save" button sequence, because the status button can now possibly be a "Review & save" link, which looks nicer and more organized when placed before the "save" button.